R/agg_scale.R

```diff
@@ -57,6 +55,53 @@
 #' @return \[`data.table()`\] with `id_cols` and `value_cols` columns for
 #'   requested aggregates or with scaled values.
 #'
+#' @section Severity Arguments:
+#' **`missing_dt_severity`**:
+#'
+#' Check for missing levels of `col_stem`, the variable being aggregated or
+#' scaled over.
+#' 1. `stop`: throw error (this is the default).
+#' 2. `warning` or `message`: throw warning/message and continue with
```

Comment:
Does "where possible" mean that no aggregates will be produced for groups with missing levels? And similarly that no scaling will be done for those groups? I think the language could just be a tad clearer between "continue with aggregation/scaling where possible" and "...continue with aggregation/scaling" in the skip definition below.

Reply:
Correct. Changed to
```
#' 2. `warning` or `message`: throw warning/message and continue with
#' aggregation/scaling for requested aggregations/scalings where expected input
#' data in `dt` is available.
#' 3. `none`: don't throw error or warning, continue with aggregation/scaling
#' for requested aggregations/scalings where expected input data in `dt` is
#' available.
```